### ISSUE TITLE
Midwinter Festival Fail

### ISSUE THREAD

**Issue by Ladysarajane** [teagan75/Metamorphoses#8]
When the time came for the midwinter festival, I could not participate because of an error. All of my characters from HF had this error. 
![2019-06-14_19 43 52](https://user-images.githubusercontent.com/34258851/59551606-299cbb00-8f4a-11e9-9427-1d891c7f662f.png)

**Comment by teagan75** [comment]
This is the mod’s fault. Unfortunately no one is developing this mod anymore. You should be able to talk to the npcs once the festival is over

**Comment by Ladysarajane** [comment]
Yeah...but I have another problem regarding the "New Year's" Festival. I had my axe into the blacksmith for upgrading...now that it is spring this is what the Blacksmith says:
![2019-06-15_09 24 33](https://user-images.githubusercontent.com/34258851/59552492-a84b2580-8f55-11e9-811f-fa25473de07b.png)

**Comment by teagan75** [comment]
I wonder if that would have happened even without the festival glitch. You
can restore a backup from before you gave the axe and then wait till spring
to give it to him.

**Comment by Ladysarajane** [comment]
LOL...that is what I am trying to do right now. Just loaded one I hope was from before I gave it to him. I will let you know how it goes. Might need a warning in Quest Book about NOT giving him items to repair before the end of year and picking them up before the end of year. I didn't have trouble with New Year's Festival. It went without a problem for me. Just the Midwinter's Festival was glitched.

**Comment by teagan75** [comment]
I don't even see that festival on the calendar. I see Starry Night and New
Years. I hope the restored back up works for you

**Comment by Ladysarajane** [comment]
Sorry, Starry Night is the Midwinter's Festival. (I'm reading a book that refers to it as Midwinter's Festival) The backup took me to Winter 3. I had to remember to reload the Quest Book because the backup was before the update. At least I don't have to start over.

**Comment by teagan75** [comment]
So it’s your previous town? Well I hope nothing bad happens

**Comment by Ladysarajane** [comment]
No not my previous town...same town just running on 3.1 version...needed to update it to 3.2. Starry Night festival didn't occur which was fine...I received the invite to it the same day of New Year's Eve festival. New Year's Festival went on without problems...Blacksmith didn't have my tool at the time and all is well now on the first day of Spring. I just gave the blacksmith my Axe and he repaired and returned with difficulty on the third day of spring.

**Comment by Ladysarajane** [comment]
I had to manually complete the copper axe quest. Since it was the item  that was involved in forcing me to use a backup, I think it was borked, but you may want to check it to make sure of it.

**Comment by teagan75** [comment]
did you use the axe and it still didn't register the quest?

**Comment by Ladysarajane** [comment]
Yes, I used it and still didn't complete

**Comment by teagan75** [comment]
Ok I set all the HF tool quests to ignore NBT data. This wills solve the
"you have to use the item for it to register" annoyance. There was a reason
I didn't do this before but it seems to have resolved itself. Will be in
next update.

**Comment by Ladysarajane** [comment]
Okay thanks....I wasn't sure if my issue was caused by NBT or the backup issue. Just thought I would report it in case it was NBT.
